onnxscript/converter.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, name: Optional[Union[str, List[str]]], kind: ConverterExpress
     def is_const(self):
         return self.kind == ConverterExpressionKind.CONST
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.name
```

Comment:
@gramalingam `name` is `Optional[Union[str, List[str]]]`. Should we change this or the type?

Reply:
Change the type to `optional[str]` ... that's what translation of `None` seems to generate

Reply:
Do you mean changing the name's type to optional[str]? `__str__` always needs to return a string. should we do something to self.name when is it None in this function?

Reply:
I guess we could do `"" if self.name is None else self.name`. Not sure if and where this is used.

Reply:
I will add an assert for now since it looks like we don't expect it to be None or anything other than str.

Reply:
I have a recollection that this may be used in the translation of `None` ... in other words, I suspect we might be creating a ConverterExpression that has no name (that is, self.name is None). However, we likely don't invoke the str method on it. So, it's fine in that sense. (Anyway, I suspect a bigger cleanup of ConverterExpression is due at some point, so that's another reason not to bother much for now.)